test/app_testing/run_app_tests.sh

```diff
@@ -1,3 +1,4 @@
 
+pip install tensorflow
 
 python test_runner.py --poc ../../nvflare/poc --n_clients 2 --yaml test_apps/test_simple.yml --app_path test_apps --cleanup
```

Comment:
Not sure if we need to fix this warning "No newline at end of file."

Reply:
I don't think formatting rules apply to sh files

Reply:
I also think it doesn't matter.